src/GraphQL/loaders/defaultGraphQLTypes.js

```diff
@@ -223,6 +222,11 @@ const DATE = new GraphQLScalarType({
   },
 });
 
+const GraphQLUpload = new GraphQLScalarType({
```

Comment:
Do you think that the type change could be a breaking change for someone with using auto or maybe customizing the schema?

Reply:
Not a breaking change, this is a dummy type. Before we use the GraphQLUpload type from the `graphql-upload` package. This package is now removed since Yoga support multipart GQL upload out of the box.